test/offer_schema.js

```diff
@@ -114,6 +114,18 @@ describe("# Offer Schema tests", () => {
                     expect(err.errors.location).toHaveProperty("message", "Path `location` is required.");
                 }
             });
+
+            test("'requirements' is required", async () => {
+                const offer = new Offer({});
+                // Returning the validation promise to ensure the test doesn't finish before all the assertions do
```

Comment:
This is why we don't usually put too many comments, they can get outdated and lose meaning.
Please dele this comment and all others in that file ( and other schema test files if they use the same structure)